types/schemas/prototypeApplication/index.ts

```diff
@@ -29,19 +28,19 @@ interface Application<T extends ApplicationType> {
 }
 
 /**
- * @internal
- * Some application types will have an extra property 'preAssessment' which contains the PlanX "result"
- * Ref https://docs.google.com/spreadsheets/d/1FgULPemnwuwysrYGEkReYFXz3n3T7W0nWziU00taZE4/edit?gid=0#gid=0
+ * @title PlanX pre-assessment
+ * @description The result of the application based on information provided by the user, prior to assessment by a planning officer
  */
-interface PlanXPreAssessment {
-  preAssessment: PreAssessment;
-}
+export type PlanXPreAssessment = {
+  preAssessment: {
+    value: string;
+    description: string;
+  }[];
+};
+
+// Types below should include "submittable" application types only, not PlanX service-level-only prefixes like 'ldc'
+// All types are exported and acronymns are spelled out in full as they'll become the top-level toggles here https://theopensystemslab.github.io/digital-planning-data-schemas-docs/
 
-/**
- * @internal
- * Types below should only include "submittable" application types only, not PlanX service-level-only prefixes like 'ldc'
- * All types are exported and acronymns are spelled out in full as they'll become the top-level toggles here https://theopensystemslab.github.io/digital-planning-data-schemas-docs/
- */
```

Comment:
This `@internal` was accidently catching the `AdvertConsent` definition below it and causing this stray `object` instead of the full type name: 
![Screenshot from 2025-05-02 11-04-30](https://github.com/user-attachments/assets/23f8afea-47e8-41cd-a5e0-f0bb8401d447)